teps/0111-propagating-workspaces.md

```diff
@@ -0,0 +1,618 @@
+---
+status: proposal
```

Comment:
```suggestion
status: implementable
```

@chitrangpatel here's the status options: https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-workflow